project1/index.html

```diff
@@ -6,67 +6,66 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
+    <title>Smart Octopus</title>
 </head>
 
 <body>
-
-    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
-
-    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
-        intelligent.</h2>
-
-    <figure>
-        <img src="images/merlin.jpg" alt="lignt pink octopus">
-        <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
-            uncertain how the trait evolved</figcaption>
+    <h1>Yes,
+        the Octopus Is Smart as Heck. But Why?</h1>
+    <h2>It has eight arms,
+        three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so intelligent.</h2>
+    <figure><img src="images/octopus.jpg" alt="octopus illustration">
+        <figcaption>An purple octopus illustration. The animals are highly intelligent,
+            but researchers are uncertain how the trait evolved</figcaption>
     </figure>
-
-    <h2><img src=https://static01.nyt.com/images/2018/06/12/multimedia/author-carl-zimmer/author-carl-zimmer-thumbLarge.png>By
-            <a href="https://www.nytimes.com/by/carl-zimmer">
-        Carl Zimmer</a></h2>
-
+    <p class="class2">Carl Zimmer </p>
```

Comment:
Review [guidelines on naming classes](https://mica-web.github.io/gd-220/hints/naming-classes.html) -- `class2` isn't informative enough that, on its own, I know what it's trying to effect or what effect it may have. Some other possibilities:
- match-headline (if the class would always be applied to match the element to the headline color
- meta-color (if the class would only be used on article meta items and to add color to them)
- author-byline (if the class would only be used to draw attention/differentiate the author byline)

None of these is _perfect_ but they each are more informative than the current classname.